utils/math/safe_math.go

```diff
@@ -15,17 +14,28 @@ import (
 var (
 	ErrOverflow  = errors.New("overflow")
 	ErrUnderflow = errors.New("underflow")
+
+	// Deprecated: Add64 is deprecated. Use Add[uint64] instead.
+	Add64 = Add[uint64]
+
+	// Deprecated: Mul64 is deprecated. Use Mul[uint64] instead.
+	Mul64 = Mul[uint64]
```

Comment:
These are used by coreth. I was too lazy to make a PR to update the dependency with this. We should remove these once coreth has removed their usage.